backend/gateway/meta/meta.go

```diff
@@ -171,3 +174,33 @@ func resolvePattern(message descriptor.Message, pattern *apiv1.Pattern) *auditv1
 
 	return &auditv1.Resource{TypeUrl: pattern.TypeUrl, Id: resourceName}
 }
+
+// APIMetadata returns the API request/response interface as an anypb.Any message.
+func APIMetadata(metadata interface{}) *any.Any {
```

Comment:
Referenced this as a guide: https://github.com/lyft/clutch/blob/main/backend/resolver/resolver.go#L50

Reply:
see comment below starting on :193